test/transaction.ts

```diff
@@ -18,12 +18,21 @@ import * as pfy from '@google-cloud/promisify';
 import * as arrify from 'arrify';
 import * as assert from 'assert';
 import * as proxyquire from 'proxyquire';
+import { google } from '../proto/datastore';
+import { RequestConfig } from '../src/request';
+import { Entity } from '../src/entity';
+import { Transaction } from '../src';
+
+//tslint:disable-next-line no-any
+type Any = any;
```

Comment:
Not a big fan of this.  It's basically just disabling `noImplicitAny`.

Reply:
Should we keep adding `tslint:disable-next-line no-any` whenever we use `any` then? Just thought it'd clean things up a bit 🤷‍♂️